### DISCUSSION TITLE
Bind VIP to ECS Instance

### DISCUSSION THREAD

**Discussion by agepi477** [opentelekomcloud/terraform-provider-opentelekomcloud#1236] [Q&A]
Hi,
I created an ECS Instance with a fixed IP address using:

`network {
    name = "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
    fixed_ip_v4 = "10.10.100.90"
    access_network = true
  }`

This worked fine. I was also able to create a new Virtual IP in the Subnet using:

`resource "opentelekomcloud_networking_vip_v2" "tf-test-vip_1" {
  network_id = "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
  subnet_id  = "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
  ip_address = "10.10.100.91"
}`

This also worked as it supposed to.

Now I struggle to bind this Virtual IP to the NIC of the ECS Instances. Could you please show me how to achieve this?

Thanks in advance.

Hendrik

**Comment by lego963** [comment]
@agepi477 if you want to associate your Virtual IP you can use `resource/opentelekomcloud_networking_vip_associate_v2`
https://registry.terraform.io/providers/opentelekomcloud/opentelekomcloud/latest/docs/resources/networking_vip_associate_v2

**Reply by agepi477** [reply]
@lego963 thanks I was able to solve this.
